internal/legacy.faustjs.org/docs/core/reference/config.mdx

```diff
@@ -22,7 +22,7 @@ if (!process.env.NEXT_PUBLIC_WORDPRESS_URL) {
  */
 export default coreConfig({
   wpUrl: process.env.NEXT_PUBLIC_WORDPRESS_URL,
-  apiClientSecret: process.env.FAUSTWP_SECRET_KEY,
```

Comment:
For backwards compatibility I think you should leave the old docs intact as the old Faust does not check the FAUST_SECRET_KEY. If the user has old version of Faust and follow this advice their builds will fail.